components/tikv_util/src/sys/cgroup.rs

```diff
@@ -58,85 +59,87 @@ pub struct CGroupSys {
 }
 
 impl CGroupSys {
-    pub fn new() -> Self {
+    pub fn new() -> Result<Self, String> {
         let lines = read_to_string("/proc/self/cgroup").unwrap();
-        let is_v2 = is_cgroup2_unified_mode();
+        let is_v2 = is_cgroup2_unified_mode()?;
```

Comment:
This change does not seem to make any difference. If this returns an error, we are using a default `CGroupSys`. Then, I think later calls like `cpuset_cores` will still panic?

Reply:
The new test case `test_defult_cgroup_sys` shows that a default `CGroupSys` will return default value for `cpuset_cores` and other methods.

Reply:
Ah, get it.